Dockerfile

```diff
@@ -1,8 +1,20 @@
-FROM gluwa/ci-linux:production AS builder
+# hadolint global ignore=DL3008,DL4006,SC2086
+
+FROM ubuntu:22.04 as builder
 ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
```

Comment:
May want to call upgrade here to ensure misc versions match between the builder and runner